source/_docs/automation/trigger.markdown

```diff
@@ -904,6 +903,12 @@ Note that to use JSON encoded payloads, the `Content-Type` header must be set to
 ```bash
 curl -X POST -H "Content-Type: application/json" -d '{ "key": "value" }' https://your-home-assistant:8123/api/webhook/some_hook_id
 ```
+If you want to pass webhook data to action, you will need to use something like this in your action
+```yaml
+data_template:
```

Comment:
There is no need to use a data_template, it features has been added years ago, we now only use the data key.